astroquery/eso/tests/test_eso_remote.py

```diff
@@ -17,10 +16,11 @@
 
 SKIP_TESTS = not(HAS_KEYRING and ESO_IMPORTED)
```

Comment:
While we at it, I wonder what is the reason to skip the test if the module is not importable (both here in the remote and in the local testing file). Sure, it cannot be imported when ``keyring`` is not available, but I think if something else go wrong it may hide undetected because of this ``try ... except``.